### ISSUE TITLE
jeepCapVideoPlayerExit Event Doesn't Always Fire When Player is Closed

### ISSUE THREAD

**Issue by epasch** [jepiqueau/capacitor-video-player#90]
On iOS, if you click the [X] to close the fullscreen player, the jeepCapVideoPlayerExit event fires. However if you swipe down on the player to close it, the event does not fire. The only event that fires when swiping to close is the jeepCapVideoPlayerPause. 

Also, when the player is closed due to the ending of the video, the jeepCapVideoPlayerExit event should fire as well as the jeepCapVideoPlayerEnded. Currently only the jeepCapVideoPlayerEnded event fires.

Basically, I need to know when that player is closed for any reason (click the close button, swipe to close, video ended, etc).

I've confirmed this behavior on an iPhone 13 pro and an iPhone 7.

**Comment by jepiqueau** [comment]
@epasch thanks for using the plugin. You cancreate in your app a general listener listening for the end and exit lustener of the plugin and you will have what you are looking for. For the swipe down i do not  know what it is fired  you mentioned the pause so i will have to look more in detail

**Comment by jepiqueau** [comment]
@epasch when you swipe down the application is not active so the video is paused but not exit. When you put  the application active again do you want the video to restart from where it was before the swipe down or another behaviour ?

**Comment by mattsputnikdigital** [comment]
@jepiqueau I have just discovered this issue on iOS as well.

I believe he is referring to when you drag the video down, it doesn't close the app, it cancels the video and the capacitor app is still running underneath.

**Comment by mattsputnikdigital** [comment]
@jepiqueau I believe if you set .isModalInPresentation = true it will stop the user from being able to swipe the video off screen. Given the video is lost this will prevent that bug, the other issue, firing an event if the video is swiped away, I do not know how to solve yet.
